lte/gateway/python/integ_tests/s1aptests/BUILD.bazel

```diff
@@ -59,6 +60,1053 @@ py_library(
     ],
 )
 
```

Comment:
It seems that the `test_enb_complete_reset` is missing in bazel?

Reply:
It is indeed, I somehow missed it. Nice find!